types/double-metaphone/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for double-metaphone 1.0
+// Project: https://github.com/words/double-metaphone
+// Definitions by: Benedikt Elssmann <https://github.com/BenediktElssmann>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Compute the double metaphone representation of a word.
+ * @param value The input word.
+ */
+declare function doubleMetaphone(value: string): any;
```

Comment:
```suggestion
declare function doubleMetaphone(value: string): [string, string];
```
or even (but this would require min TS version 4.0:
```ts
declare function doubleMetaphone(value: string): [primary: string, secondary: string];
```
it just cannot be any

Reply:
I totally agree and tried to use `string[]` first, but running `npm test` I got the error `ERROR: 3:7  expect  TypeScript@4.3 compile error: 
Type 'string[]' is not assignable to type 'string'.` (almost the same error using `[string, string]`)

Reply:
It's not a string[], it's a tuple:
```ts
declare function doubleMetaphone(value: string): [string, string];
```
I've just tested this DT with compiled code and it works a treat, you've probably got some staled content locally when testing.